_schemaV2.graphql

```diff
@@ -8106,13 +8106,22 @@ type Partner implements Node {
   # A connection of artworks from a Partner.
   artworksConnection(
     after: String
+
+    # Return only artwork(s) included in this list of IDs.
+    artworkID: [String]
```

Comment:
should this be `artworkIDs`?

Reply:
@annacarey yeah I hear you on this 😅 we did this as `artworkID` because that's what it is in the Gravity endpoint: https://github.com/artsy/gravity/blob/dd5fd7149177b26b138c6d7fd1f27cf3aed3f13d/app/api/v1/partners_artworks_endpoint.rb#L96

That said, we don't _have_ to follow that if we don't want to. My thinking here was to prioritize consistency, but it definitely does read weird. I don't think it's too confusing if this is plural and the gravity version is singular, I'll make that adjustment